mailer.js

```diff
@@ -26,6 +27,31 @@ module.exports = function (log) {
     return 'href="' + url + '" style="color: #0095dd; text-decoration: none; font-family: sans-serif;"'
   }
 
+  function constructLocationString (location) {
+    // construct the location string from the location object
+    var locationString = ''
+    if (location) {
+      if (location.city) {
+        locationString += location.city + ', '
+      }
+      locationString += location.country + ' ' + '(estimated)'
+    }
+    return locationString
+  }
+
+  function constructLocalTimeString (timeZone, locale) {
+    // if no timeZone is passed, use 'Etc/GMT'
+    moment.tz.setDefault('Etc/GMT')
```

Comment:
@shane-tomlinson Whats the difference? :O IANA says both are equivalent :confused: